helpers/integration.go

```diff
@@ -51,6 +52,40 @@ type HACBSTestResult struct {
 	Warnings  int    `json:"warnings"`
 }
 
+var testSchema = `{
```

Comment:
Maybe `testResultSchema` ?

Reply:
updated naming to `testResultSchema` in most recent PR